test/integration/only_admin_can_view_admin_dashboard_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.